mne/io/meas_info.py

```diff
@@ -267,6 +262,24 @@ def _update_redundant(self):
         self['nchan'] = len(self['chs'])
 
 
+def anonymize_info(info):
+    """Anonymize measurement info in place.
+
+    Parameters
+    ----------
+        info : instance of Info
+        Measurement information for the dataset.
+    """
+    if not isinstance(info, Info):
+        raise ValueError('info must be an Info instance.')
+    if info.get('subject_info') is not None:
+        del info['subject_info']
+    info['meas_date'] = [0, 0]
+    for key_1 in ('file_id', 'meas_id'):
+        for key_2 in ('secs', 'msecs', 'usecs'):
+            info[key_1][key_2] = 0
```

Comment:
also, why break this out into a function instead of just making the `info._anonymize` public?